### DISCUSSION TITLE
API Security Project Discussion

### DISCUSSION THREAD

**Discussion by ErezYalon** [OWASP/API-Security#120] [Announcements]
## 👋 Welcome to the OWASP API Top Ten community!
  We’re using Discussions as a place to connect with other community members.
  Feel free to:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and be open-minded.
  * Remember that this is a community we build together 💪.
  * Comment below with an introduction of yourself and tell us about what you do with APIs.

**Comment by irisleva** [comment]
this https://github.com/OWASP/API-Security/blob/master/editions/2023/en/0x10-api-security-risks.md) is supposed to contain API10 but it doesn't.
the correct text is in the  https://owasp.org/API-Security/editions/2023/en/0xaa-unsafe-consumption-of-apis/

**Reply by PauloASilva** [reply]
Hi @irisleva,

> this https://github.com/OWASP/API-Security/blob/master/editions/2023/en/0x10-api-security-risks.md) is supposed to contain API10 but it doesn't.

Why do you assume that?

If you check previous releases (e.g. [v2.1.1] or [v0.1.0]) you'll realize it is like that since the beginning.
You may not like the file names but I believe there's nothing wrong here. You'll find the Top 10 categories from `0xa1` to `0xaa` ;)

[v2.1.1]: https://github.com/OWASP/API-Security/blob/v2.1.1/2019/en/src/0x10-api-security-risks.md
[v0.1.0]: https://github.com/OWASP/API-Security/blob/v0.1.0/2019/en/src/0x10-api-security-risks.md

Cheers,
Paulo A. Silva
